### DISCUSSION TITLE
Linglong discussion

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3372] [Q&A | 问答板块]
Hello, I've seen that deepin is in the process of making new packaging system. I'm curious about some aspects of it. 

1. Will Linglong support installing applications from archive like files? Similarly to .apk and .exe? 
2. Suggestion - If it's still feasible you can shorten **Linglong** to **Lilo** for command line operations, 
a) it would stop making confustion with `ll` which is used by a lot of people to perform listing, 
b) it would be a more distinctive name
c) I think it would be a better name for English speakers in general. 

Lilo used to be a name for linux boot loader, but it has been depreciated years ago. From what I've check the name seems to be unoccupied.

**Comment by BLumia** [comment]
The whole thing is not open-sourced yet but you can already try it on its related website <https://linglong.space/>. I am not a part of linglong's development team so not be able to answer all related questions, but if you visit that site and maybe try the thing, you may get some answer of your questions already :)

> Will Linglong support installing applications from archive like files? Similarly to .apk and .exe?

It's a new package format, not an emulator/emulation layer. It can be used to run a emulator tho.

> If it's still feasible you can shorten Linglong to Lilo for command line operations,

The default executable is not `Linglong` nor `ll`, it's `ll-cli`, and you can always create a preferred bash alias by yourself :)
